integration/kubernetes/runtimeclass_workloads/pod-quota-deployment.yaml

```diff
@@ -11,7 +11,7 @@ spec:
   selector:
     matchLabels:
       purpose: quota-demo
-  replicas: 3
+  replicas: 2
```

Comment:
why this change? Is it related to not leaving pods around?

Reply:
@grahamwhaley yes, with this we will not leave pods

Reply:
but... why? Can you explain (maybe in the commit message) why 3->2 makes it not leak?

Reply:
@GabyCT can you open an issue in the runtime repo and add it as a reference maybe in the commit message?

Reply:
@chavafg and @grahamwhaley changes applied